py/test_parse_and_notify.py

```diff
@@ -49,42 +49,46 @@ def test_markdown_table(
 
 
 def test_topicctl() -> None:
-    content = {
```

Comment:
since we're doing 1 line per topic, that has an implicit assumption we'll only have one of `newTopic` or `updatedTopic` filled in per line so I broke this up into `new_topic_content` and `update_topic_content`